### DISCUSSION TITLE
Scraping Tools Not Working On Render

### DISCUSSION THREAD

**Discussion by cjaysd** [FlowiseAI/Flowise#779] [Q&A]
When run locally, the scraping tools work flawlessly. Has anyone had any luck getting these to run on a Render hosted instance?

**Comment by chungyau97** [comment]
Hi @cjaysd,

Could you share out the error?

**Comment by Charlotte-br560** [comment]
Certainly! When facing issues with scraping tools on Render, it's essential to ensure that all dependencies are properly configured for the hosted environment. You might find it beneficial to explore how similar challenges have been addressed in projects like [Crawlbase](https://crawlbase.com/). Their documentation or community discussions could offer insights into optimizing scraping tools for successful execution on Render-hosted instances. Good luck with resolving the compatibility concerns!
